gold-cc-cvc-input.html

```diff
@@ -199,11 +199,10 @@
          */
         validate: function() {
           // Empty, non-required input is valid.
-          if (!this.required && this.value == '') {
-            return true;
-          }
-
-          var valid = this.value.length == this._requiredLength;
+          var valid =
+              (this.$.input.validate() &&
+              this.value.length == this._requiredLength) ||
+              !this.required && this.value == '';
```

Comment:
The problem with removing it is that the length check should only apply when `this.required`. The `||` lets us fail the length check but still be valid.

I could actually get rid of both the length and required checks by setting minlength on the input field, but support for minlength is pretty spotty (terrible) outside of chrome, so not really an option. Having said that, it's used in paper-input, but I think that's just for delegation to the inner input, rather than any belief that it's actually going to do anything sensible.